lib/backend/k8s/k8s.go

```diff
@@ -214,6 +219,29 @@ func (c *KubeClient) getProfile(k model.ProfileKey) (*model.KVPair, error) {
 	return c.converter.namespaceToProfile(namespace)
 }
 
+// applyWorkloadEndpoint patches the existing Pod to include an IP address, if
+// one has been set on the workload endpoint.
+func (c *KubeClient) applyWorkloadEndpoint(k *model.KVPair) (*model.KVPair, error) {
+	ips := k.Value.(*model.WorkloadEndpoint).IPv4Nets
+	if len(ips) > 0 {
+		log.Debugf("Applying workload with IPs: %+v", ips)
+		ns, name := c.converter.parseWorkloadID(k.Key.(model.WorkloadEndpointKey).WorkloadID)
+		pod, err := c.clientSet.Pods(ns).Get(name)
+		if err != nil {
+			return nil, err
+		}
+		pod.Status.PodIP = ips[0].IP.String()
+		log.Debugf("Pod IP: %+v", pod.Status.PodIP)
+		pod, err = c.clientSet.Pods(ns).UpdateStatus(pod)
```

Comment:
The k8s API won't care if we're setting this field on something that already has an address.

We actually want to change it anyway - if an IP already exists it means the pod has restarted and might be getting a new IP address.